CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Added manifest validation error.
 
+### Security
+
+- Including helm upstream security fix with [v3.1.3](https://github.com/helm/helm/releases/tag/v3.1.3).
```

Comment:
```suggestion
- Updated Helm to [v3.1.3](https://github.com/helm/helm/releases/tag/v3.1.3) for security fix.
```

Good to put this under security 👍

I find writing changelogs hard but what about this? I think it's a bit shorter and so easier to read quickly.

Reply:
Thanks for pointing out. I adjust it as your suggestion.